runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/ResumeFromCheckpointStreamingTest.java

```diff
@@ -102,11 +101,9 @@
   private static final String TOPIC = "kafka_beam_test_topic";
 
   @Rule
-  public TemporaryFolder tmpFolder = new TemporaryFolder();
+  public final transient ReuseSparkContextRule noContextResue = ReuseSparkContextRule.no();
```

Comment:
This could be a separate ticket: "enhance PipelineRule". 
Should be doable since it uses `RuleChain`